pylint/lint.py

```diff
@@ -1227,7 +1137,35 @@ def get_ast(self, filepath, modname):
             self.add_message("astroid-error", args=(ex.__class__, ex))
 
     def check_astroid_module(self, ast_node, walker, rawcheckers, tokencheckers):
-        """Check a module from its astroid representation."""
+        """Check a module from its astroid representation.
+
+        For return value see _check_astroid_module
+        """
+        before_check_statements = walker.nbstatements
+
+        retval = self._check_astroid_module(
+            ast_node, walker, rawcheckers, tokencheckers
+        )
+
+        self.stats["by_module"][self.current_name]["statement"] = (
+            walker.nbstatements - before_check_statements
+        )
+
+        return retval
+
+    def _check_astroid_module(self, ast_node, walker, rawcheckers, tokencheckers):
+        """Check given AST node with given walker and checkers
+
+        ast_mode: AST node of the module to check
```

Comment:
Can we switch to restructuredText instead? It's most used structured documentation format used in the codebase.

Reply:
done